src/main/java/org/folio/rest/impl/NotesResourceImpl.java

```diff
@@ -312,37 +307,32 @@ private void pn5InsertNote(Context context, String tenantId, Note note,
    *
    * @param id
    * @param okapiHeaders
-   * @param resp a callback that returns the note, or an error
    */
-  private void getOneNote(String id, Map<String, String> okapiHeaders,
-    Context context, Handler<ExtendedAsyncResult<Note>> resp) {
+  private CompletableFuture<Note> getOneNote(String id, Map<String, String> okapiHeaders, Context context) {
```

Comment:
since we work with pg client and it uses vertex futures, let's stick with them and do not use java futures

Reply:
you can leave it as is for now
the issue will be addressed in other PR